app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java

```diff
@@ -16,7 +20,24 @@ public ParkingView(Activity activity, ActivityMainBinding binding) {
     }
 
     @Override
-    public void displayAvailableParkingSpots(String parkingSpots){
-        binding.parkingTextView.setText(getActivity().getString(R.string.activity_main_available_text) + parkingSpots);
+    public void displayAvailableParkingSpots(String parkingSpots) {
+        binding.parkingTextView.setText(getActivity().getString(R.string.activity_main_available_text, parkingSpots));
+    }
+
+    @Override
+    public void displayFragment(ListenerDialogFragment listener) {
+        ParkingFragment fragment = ParkingFragment.newInstance(listener);
+        FragmentManager fragmentManager = getFragmentManager();
+        fragment.show(fragmentManager, ParkingFragment.PARKING_FRAGMENT_TAG);
+    }
+
+    @Override
+    public void displayToastExceededInt() {
+        Toast.makeText(binding.getRoot().getContext(), getActivity().getString(R.string.toast_exceeded_int), Toast.LENGTH_LONG).show();
```

Comment:
You shouldn't use this getRoot here,  it is not necesary. You should use just getContext (remind to test if this context isn't null)

Reply:
fixed